RELEASENOTES.md

```diff
@@ -3,6 +3,12 @@
 * PR #54 Move MersenneTwister to built-in C++ function.
 	* The software now requires C++ 11 or newer.
 * PR #55 fixed sphinx documentation warnings. 
+* PR #56 Travis now checks if files are empty.
+* PR #? Improvements to documentation:
```

Comment:
Maybe reference the issue this is in reference to. I guess I forgot to. do that too in the line above. Also I think the PR is #58

Reply:
Included PR # and added issue references for everything in this section.